github.ghactivity.php

```diff
@@ -240,6 +240,84 @@ public function get_github_issues( $repo_name, $page_number = 1 ) {
 		return $this->get_github_data( $query_url );
 	}
 
+	/**
+	 * Remote call to get all org projects
+	 *
+	 * @since 2.1.0
+	 *
+	 * @param string $org_name   name of the repo.
+	 * @param int    $page_number page number of paginatated response.
+	 *
+	 * @return array
+	 */
+	public function get_projects( $org_name, $page_number = 1 ) {
+		$headers = array( 'accept' => 'application/vnd.github.inertia-preview+json' );
```

Comment:
Are those headers necessary for all GitHub requests? Can they be different? If they are always the same, maybe we should just add them to `get_github_data` so it gets applied to all requests right away?

Reply:
Projects API still in beta, which means it might be unstable. To be able to use these endpoints above header should be passed. without that header endpoints will return errors.

I don't think we should add this header to `get_github_data` since it will affect all endpoints. I will extract that header into separate function for clarity